### ISSUE TITLE
death rates for DT2 bosses are absurdly high (NOT Awakened!!!)

### ISSUE THREAD

**Issue by mylife212** [oldschoolgg/oldschoolbot#5279]
## 🚀 Feature Proposal

The death rate for the DT2 bosses is like 95% when you're 0 kc and scales down very slowly until 100 kills. This is not accurate to ingame, where you do die when learning, but not nearly as often. I'd say a learning curve of: 40% death rate per boss at 0 kc, scaling down 5% for every 10 kills until it caps at 5% death rate around 40kc is much more fair. Right now, you die 15 times in 1 duke trip lol

tldr: reduce death rate for normal bosses

However, the death rate for awakened bosses are perfect. This is for normal variants.

**Comment by gc** [comment]
Done in next update,

before: https://cdn.discordapp.com/attachments/940760682561929259/1144691044974350498/file.jpg

after: https://cdn.discordapp.com/attachments/940760682561929259/1144690850765492233/file.jpg
